### ISSUE TITLE
Black dot on the left side of titlebar and taskbar in Windows

### ISSUE THREAD

**Issue by helloanoop** [usebruno/bruno#179]
![image](https://github.com/usebruno/bruno/assets/9350733/0cbddc30-a118-4381-a6c2-a5873a2e6e72)
![image (1)](https://github.com/usebruno/bruno/assets/9350733/30984f03-f5d2-490a-bfe9-d2e210c8cb0f)

**Comment by helloanoop** [comment]
As per ChatGPT

The black dot that appears to the left of an app title in an Electron build for Windows is typically the default icon used for Electron applications when you haven't specified a custom icon. This dot is a placeholder icon and is used as a fallback when no custom icon is provided.

To change this icon and replace it with a custom one, you can follow these steps:

Create or Obtain an Icon: First, create or obtain an icon in the .ico format that you want to use as your application's icon. Ensure that the icon is designed in multiple sizes to accommodate different resolutions (e.g., 16x16, 32x32, 64x64, 256x256).

Specify the Icon in Your Electron App: In your Electron application's main script (usually the main.js or main.ts file), you can set the application icon using the app module. Import the app module if you haven't already and use the setIcon method to set the icon.

```js
const { app, BrowserWindow } = require('electron');
const path = require('path');

// Specify the path to your custom icon file
const iconPath = path.join(__dirname, 'path-to-your-icon.ico');

// Set the application icon
app.whenReady().then(() => {
  app.setIcon(iconPath);
});

// Create your main BrowserWindow as usual
// ...

```

**Comment by not-known-person** [comment]
do u want us to set bruno logo as application icon ,

**Comment by helloanoop** [comment]
@not-known-person Yes!

**Comment by not-known-person** [comment]
Ok , Thanks for the confirmation

**Comment by not-known-person** [comment]
hey @helloanoop , I did a research on it , and I found that the code provided by the ChatGPT is correct , but it's not a recommended approach to change the icon , Please have a look in official docs  - https://www.electronforge.io/guides/create-and-add-icons

**Comment by razzivofficial** [comment]
I think this issue is resolved @helloanoop can close this issue

**Comment by helloanoop** [comment]
Yes, Thank you for the fix @zyrouge https://github.com/usebruno/bruno/pull/393

This is available in `v0.20.0` onwards
